### PR TITLE
priveleges needed for posting in request_membership

### PR DESCRIPTION
**Pull request by ancafarcas**

### CODE REVIEW

File: server/liveblog/blogs/request_membership.py [hunk 1]
```diff
@@ -72,17 +71,19 @@ class MembershipResource(Resource):
...
-class MembershipService(BlogService):
-    notification_key = 'request'
 
+class MembershipService(BaseService):
+    notification_key = 'request'
+  
     def on_create(self, docs):
         for doc in docs:
+            doc['original_creator'] = str(get_user().get('_id'))
             doc['message'] = "Please add me as a contributor to your blog"
```
Comment (vied12) [inline server/liveblog/blogs/request_membership.py]: Do we really need to define the message here ? where is it used ?

Comment (ancafarcas) [inline server/liveblog/blogs/request_membership.py]: It isn't really needed. I think it can be deleted if @vladnicoara says it doesn't need it

Comment (vladnicoara) [inline server/liveblog/blogs/request_membership.py]: if you are just using that to let front-end know what kind of request it is and we only get one kind of request from that endpoint, I can do without it

Comment (ancafarcas) [inline server/liveblog/blogs/request_membership.py]: ok, removed it is, then!

**Pull request merged**